wiki/Beatmapping/Health/en.md

```diff
@@ -0,0 +1,101 @@
+
+Health recovery:
+
+- A successful tap on the hit circle (better judgement leads to more health gain; the last hit circle in a colour combo awards more health).
+- A successful tap on a slider start.
+- The slider ball collected a slider tick successfully.
+- The slider ball touched the slider end or a reverse arrow successfully (better judgement at the end tick leads to more health gain).
+- Completing a spinner and continuing to spin (better judgement at the end of the spinner leads to more health gain).
+
+Health loss:
+
+- Health drain.
+- Missing a hit circle.
+- Tapping a slider head early.
+- Missing a slider tick, or slider reverse arrow.
+- Failed to achieve at least a 50 on a spinner.
+
+Health reserved:
+
+- Break time.
+
+## osu!taiko
+
+In osu!taiko the health bar starts off empty and must be filled 50% or more to pass a beatmap. This is done by tapping the correct combination of keys in sync with the beat.
+
+Score bursts will appear after hitting notes, changing score and health values depending on the accuracy of the hit. A perfectly timed hit (300) will recover more health than a badly timed hit (50). Players also receive larger boosts if they achieve a [katu](#katu) or a [geki](#geki) at the end of a combo. A miss will drain health instead. Failing to complete a denden will result in health loss. Drumrolls can be ignored with no health penalty as they have no influence on the health bar. For an overview of health recovery and health loss see below:
+
+Health recovery:
+
+- Getting 'Great' or 'Good' on small/large notes.
+
+Health loss:
+
+- Getting 'Miss' or 'Bad' on small/large notes.
+- Not completing Dendens.
+- Not completing Shakers.
+
+No effect:
+
+- Doing drumrolls.
+- Ignoring drumrolls.
+- Completing Dendens.
+- Completing Shakers.
+
+_(Note: It is entirely possible for a beatmap to never pass by design due to it only containing drumrolls and/or dendens/shakers.)_
+
+## osu!catch
+
+In osu!standard health will decrease at a steady rate called [health drain](/wiki/Beatmapping/Health_drain), but can be replenished by catching fruits and drops.
+
+Score bursts will appear after catching fruit, changing score and health values depending on the hit object caught. The player also receives larger boosts if he catches the end of a combo resulting in a [geki](#geki). Missing fruit will drain health instead. Bananas can be ignored with no health penalty. For an overview of health recovery and health loss see below:
+
+Health recovery:
+
+- Collecting any fruits, juice drops, or bananas.
+
+Health loss:
+
+- Health drain.
+- Missing fruits or juice drops.
+
+Health reserved:
+
+- Break time.
+
+## osu!mania
+
+In osu!mania only hit objects will affect the health bar, as [health drain](/wiki/Beatmapping/Health_drain) is disabled.
```

Comment:
Note: health drain is not disabled in lazer.

Reply:
It makes no sense to start detailing lazer when it is unfinished. will require too much upkeep as lazer is changed

Reply:
i'd also expect game mechanics on ranked maps to eventually match osu stable